migrations/20200927205755_user.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const {tableName} = require('../src/models/user.js')
+
+exports.up = function (knex) {
+    return knex.schema.createTable(tableName, (table) => {
+        table.increments('id').primary();
+        table.text('email').notNullable().unique();
+        table.text('passwordHash').notNullable().unique();
+        table.text('firstName');
+        table.text('lastName');
+        table.text('occupation');
+        table.timestamps(true, true);
+    })
+}
+
+
+exports.down = function (knex) {
+    return knex.schema.dropTable(tableName);
+}
```

Comment:
Ideally this file should be in this PR https://github.com/TechLadies/onesg-app/pull/3/commits ..and the linter PR should have only linter related code

So, 
* let's revert the linter commit from the previous PR
* clean up this branch to have only the last commit